pkg/sync/service.go

```diff
@@ -380,6 +381,7 @@ func (s *Service) doDownloadFromURL(request *http.Request) (err error) {
 func (s *Service) UploadFromRequest(writer http.ResponseWriter, request *http.Request) {
 	err := s.doUploadFromRequest(request)
 	if err != nil {
+		err = errors.Wrapf(err, "please make sure the file is qcow2 or raw image")
```

Comment:
Is this message necessary? It misleads users that the failure is caused by incorrect file format but actually there are other kinds of errors.

Reply:
Yes you are right, put it back to the original error